doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -576,6 +257,238 @@ Field         | Type                          | Meaning
 `details`     | [`jsonb`]                     | Additional metadata provided by the sink. In case of error, may contain a `hint` field with helpful suggestions.
 
 
+## Replica Introspection Relations
+
+This section lists the available replica introspection relations.
+
+Introspection relations are maintained by independently collecting internal logging information within each of the replicas of a cluster.
+Thus, in a multi-replica cluster, queries to these relations need to be directed to a specific replica by issuing the command `SET cluster_replica = <replica_name>`.
+Note that once this command is issued, all subsequent `SELECT` queries, for introspection relations or not, will be directed to the targeted replica.
+Replica targeting can be cancelled by issuing the command `RESET cluster_replica`.
+
+For each of the below introspection relations, there exists also a variant with a `_per_worker` name suffix.
+Per-worker relations expose the same data as their global counterparts, but have an extra `worker_id` column that splits the information by Timely Dataflow worker.
```

Comment:
Do you feel this is sufficient documentation of the `_per_worker` relations? We initially discussed having a collapsible sub-section under each of the global relations, but I found a couple issues with that:
* We don't have collapsible section elements currently (AFAICS).
* The `_per_worker` schema would just copy all of the fields of the global schema and add a `worker_id` field. So there would be a lot of redundant information and keeping it in sync might be painful.